jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -1667,20 +1671,67 @@ static MethodHandle filterArgument(MethodHandle target, int pos, MethodHandle fi
 		return Lookup.PUBLIC_LOOKUP;
 	}
 	
-	/*[IF Sidecar19-SE]*/
+	/*[IF Sidecar19-B175]*/
 	/**
-	 * Return a MethodHandles.Lookup object that is only able to access <code>private</code> members.
+	 * Return a MethodHandles.Lookup object with full capabilities including the access 
+	 * to the <code>private</code> members in the requested class
 	 * 
 	 * @param targetClass - the requested class containing private members
 	 * @param callerLookup - a Lookup object specific to the caller
 	 * @return a MethodHandles.Lookup object with private access to the requested class
 	 * @throws IllegalArgumentException - if the requested Class is a primitive type or an array class
-	 * @throws NullPointerException - if any of the arguments are null
+	 * @throws NullPointerException - if any of the arguments is null
 	 * @throws IllegalAccessException - if access checking fails
 	 * @throws SecurityException - if the SecurityManager prevents access
 	 */
 	public static MethodHandles.Lookup privateLookupIn(Class<?> targetClass, MethodHandles.Lookup callerLookup) throws IllegalArgumentException, NullPointerException, IllegalAccessException, SecurityException {
-		throw new UnsupportedOperationException("The method has not yet been implemented for now"); //$NON-NLS-1$
+		if ((null == targetClass) || (null == callerLookup)) {
+			/*[MSG "K065S", "Both the requested class and the caller lookup must be non-null"]*/
+			System.out.println("NullPointerException K065S: Both the requested class and the caller lookup must be non-null"); //$NON-NLS-1$
```

Comment:
Remove the System.out.println call.

Reply:
These printing messages were used for test and already removed.